app/controllers/external_protocols_controller.rb

```diff
@@ -9,12 +9,12 @@ def index
     service_call = ProtocolImporters::SearchProtocolsService
                    .call(protocol_source: index_params[:protocol_source],
                          query_params: index_params)
-
     if service_call.succeed?
+      show_import_button = current_user.is_normal_user_or_admin_of_team?(current_team)
```

Comment:
Better to use `can_create_protocols_in_repository?(@team)` to be consistent.  See `check_import_permissions` method.

Reply:
groovy, tnx